docs/arcaflow/creating-plugins/python.md

```diff
@@ -10,151 +10,50 @@ The easiest way is to start from the [template repository for Python plugins](ht
-           ```
-           # yaml-language-server: $schema=example.schema.json
-           ```    
-    11. Copy and customize the [Dockerfile](https://github.com/arcalot/arcaflow-plugin-template-python/blob/main/Dockerfile) from the example repository.
-    12. Set up your CI/CD system as you see fit.
-
-=== "Using Poetry"
-
-    1. Assuming you have [Poetry](https://python-poetry.org) installed, run the following command:
-           ```
-           poetry new your-plugin
-           ```
-       Then change the current directory to `your-plugin`.
-    2. Figure out what the right command to call your Python version is:
-           ```
-           which python3.10
-           which python3.9
-           which python3
-           which python
-           ```
-       Make sure you have at least Python 3.9.
-    3. Set Poetry to Python 3.9:
-           ```
-           poetry env use /path/to/your/python3.9
-           ```
-    4. Check that your `pyproject.toml` file has the following lines:
-           ```toml
-           [tool.poetry.dependencies]
-           python = "^3.9"
-           ```
-    4. Add the SDK as a dependency:
-           ```
-           poetry add arcaflow-plugin-sdk
-           ```
-    5. Copy the [example plugin](https://github.com/arcalot/arcaflow-plugin-template-python/blob/main/example_plugin.py), [example config](https://github.com/arcalot/arcaflow-plugin-template-python/blob/main/example.yaml) and the [tests](https://github.com/arcalot/arcaflow-plugin-template-python/blob/main/test_example_plugin.py) to your directory.
-    6. Activate the venv:
-           ```
-           poetry shell
-           ```
-    7. Run the test plugin:
-           ```
-           ./example_plugin.py -f example.yaml
-           ```
-    8. Run the unit tests:
-           ```
-           ./test_example_plugin.py
-           ```
-    9. Generate a JSON schema:
-           ```
-           ./example_plugin.py --json-schema input >example.schema.json
-           ```
-      If you are using the [YAML plugin for VSCode](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml), add the following line to the top of your config file for code completion:
-           ```
-           # yaml-language-server: $schema=example.schema.json
-           ```
-    10. Copy and customize the [Dockerfile](https://github.com/arcalot/arcaflow-plugin-template-python/blob/main/Dockerfile) from the example repository.
-    11. Set up your CI/CD system as you see fit.
-
-Now you are ready to start hacking away at your plugin!
+## Plugin Setup Quickstart
+
+### From the template repository
+
+1. Fork, then clone the [template repository](https://github.com/arcalot/arcaflow-plugin-template-python)
+
+2. Figure out the right command to call your Python version:
+     ```
+     python3.10 --version
+     python3.9 --version
+     python3 --version
+     python --version
+     ```
+ Make sure you have at least Python 3.9.
```

Comment:
This text is rendering as outside of the `2.` item here and between it and the `3.` item.